### DISCUSSION TITLE
Heroic/Raid instances will not reset if server is off during the reset time, is it Blizzlike or designed in purpose?

### DISCUSSION THREAD

**Discussion by wangsiw1** [azerothcore/azerothcore-wotlk#16983] [General]
I setup the server on my pc and start it whenever I am going to play and shut it down when I am done. With this I am missing the default instance reset time most of the time. Next time I start the server, all instances that missed previous reset time will show the login time as next reset time. Second start of server will correct the time with real reset time. 

For now I am manually deleting instance row in the database before starting the server, although just learned from another discussion that there is command to unbind the instances. But I would like to learn if this is intended behavior or something only happens to me?

**Comment by PhazonStorm** [comment]
Really late reply, but I also just discovered this through normal play, and after looking around it seems this is just inherent to AC and does happen to everyone. if the server isn't on it just rolls over to the next reset time.

'.instance unbind <mapid\all>' should do the trick for the reset.
